packages/generator/package.json

```diff
@@ -16,7 +16,7 @@
     "build": "npm run clean && tsc",
     "clean": "rimraf dist",
     "compare": "ts-node src/compare.ts",
-    "docs": "ts-node src/docs.ts && cd ../.. && patch -p1 apply src/ONC-compliance-text.patch",
+    "docs": "ts-node src/docs.ts && cd ../.. && git apply packages/generator/src/ONC-compliance-text.patch",
     "fhirtypes": "ts-node src/index.ts",
     "generate": "npm run fhirtypes && npm run jsonschema && npm run baseschema && npm run mockclient",
```

Comment:
Maybe we should add `npm run docs` to the end of the `generate` command? It would keep docs in sync going forward